### DISCUSSION TITLE
Again running Pi on batterie

### DISCUSSION THREAD

**Discussion by HGFestl** [mcguirepr89/BirdNET-Pi#569] [Q&A]
Hi, I am new in this community, and I like to start a n batterie discussion again. I want to run several Pi´s for bird monitoring in a forest area, if they are available again. I tested a 4B and a 0W2, and also if I disbale all the non necessary services, with a 20 000 mAh USB batterie the run time is not much more than two days. Does anybody have experiences with long time outdoor recording and a suitable power supply, possibly running about one week?
Thank you in advance, Hans Guenter

**Comment by lloydbayley** [comment]
I have been thinking about this too for a while. I was thinking of a small solar panel, a charge-controller and SLA battery. I haven't been any further than THINKING about it 'though, so would be interested in your progress.

**Comment by DCS17** [comment]
Whilst I don't use it for BirdPiNet, I do have a Zero2W running from a 12V lead/acid car battery connected to solar panels running 24/7 for 18 months now without issue. (Use it for ADSB monitoring and needs to be on top of a hill!). 
I've got a 110aH 12v leisure battery with a couple of 100w solar panels attached (I'm in cloudy UK so you may not 200W to keep up with the Zero2W consumption). Then a 12v to USB converter. Viola.
All I'm uncertain about is whether BirdNetPi might need more processing power than the Zero2W can provide? if BirdNet will run on Zero2W OK then pretty simple to set up really. But not cheap!

**Reply by HGFestl** [reply]
Thank you, DCS17! 
I have a 0W2 running with BirdNET.
Do you have something like a low voltage controll observation to protect the batterie from damage?

**Reply by DCS17** [reply]
I just used a solar kit from Amazon which had a charge controller with it.
[Amazon Solar Kit](https://www.amazon.co.uk/gp/product/B085XYMZ7S/ref=ppx_yo_dt_b_asin_title_o00_s00?ie=UTF8&th=1)
It's worked well for me, but I had to get a second panel for mid-winter short/cloudy days :-(
Got a 12v to USB converter to give USB output for the Pi
[USB Converter](https://www.amazon.co.uk/gp/product/B06XSCCLCD/ref=ppx_od_dt_b_asin_title_s01?ie=UTF8&psc=1)

There may be cheaper/better ways of doing it. But this was the simplest for me!

ps
I've just noticed the horrific inflation - I paid £98 in Sept 2020. 
2 Years later they are £150!!

**Comment by w0mbatian** [comment]
If there is a source of power within 100 metres, have you considered using power over ethernet to power the Pi 4?
This can then double as the network connection.
Ian

**Reply by HGFestl** [reply]
Unfortunately, for my application I have longer distances without power sources nearby.

**Reply by unosonic** [reply]
Use a car battery, a DC/DC step down converter and a Witty Pi 3 Mini power along with your Zero. Then you can even extend your lifetime by make suitable schedules, eg. only early mornings etc.
https://www.uugear.com/product/witty-pi-3-mini-realtime-clock-and-power-management-for-raspberry-pi/
That board can monitor an external power source and take action on a low voltage level.

**Reply by HGFestl** [reply]
Thank you, unosonic. The Witty Pi might be a nice option for some applications, but we need a 24 hour monitoring to catch especially the rare birds, and the night activities as well.

**Comment by lloydbayley** [comment]
I've been messing about making up a cable for the Raspberry Pi 4 and using one of [these](https://core-electronics.com.au/dc-dc-adjustable-step-down-module-5a-75w.html) to step the voltage down from my 12v battery that is hooked up to solar panels via a charge-controller. I've put a USB-C on the end of it which powers the Pi quite nicely. Am going to do some testing but we've had a wet weekend forecast, so I don't know how effective it will be as yet. More testing required but the overall idea is that a SLA battery, charge controller, solar panel and custom lead with step-down buck converter might just be the shot.

**Reply by HGFestl** [reply]
Keep me informed about your results! But, there are solar charge controller available, which include the USB interface. Why do you not use this?

**Reply by lloydbayley** [reply]
Because they usually don't provide a decent amount of current. Most are limited to 2A and I wanted to make sure I was replicating the PSU with comes with the Rpi4, which is rated at 5.1v, 3A. Also, the voltage in them is a bit suspect in these things for high-demand things and I've seen it drop a lot with a load. This way, you have supreme control over both.

**Reply by HGFestl** [reply]
Thank you, you are right! May be this can be used with the 0W2.

**Reply by lloydbayley** [reply]
From memory, the PSU for that was rated at about 2.5A, so yes....why not?! I can't see a 0W2 using 2.5A under normal load.

**Reply by lloydbayley** [reply]
Well, this afternoon, I cut the Pi over to the solar power. It's a bit of a test, it's been a bit cloudy and the system I'm testing it on is a smaller 18Ah battery. However, one must start somewhere. Keeping an eye on it. Hopefully it will last the night. If it does, I will make some notes on voltage drain etc. It's currently pulling about 200mA (about 3W).

**Reply by lloydbayley** [reply]
Well, I'm pleased to report that it made it overnight in overcast weather. I'm going to leave it and see how it goes over the next few days or so but so-far it's quite pleasing. I think the battery dropped to about 46% but I think that figure might be a little lower than it actually was. The MPPT controller is also datalogging charge/discharge amps, watts and volts, so I will analyse the graphs after a few days.

**Reply by HGFestl** [reply]
Thank you for this update. Did you disable any process or LED on the Pi? Does it run any applcation?

**Reply by lloydbayley** [reply]
I have not made any reduction modifications, in-fact, it runs a nightly backup to my NAS drive as well as some telemetry monitoring, so it's actually running a tiny bit more than normal.

**Reply by lloydbayley** [reply]
I am taking a break from the forums. Good luck with your battery exploits. My system is still up and running with the specs I provided earlier.

**Comment by lloydbayley** [comment]
I'm back. Update: It is still running on batteries but I had to change it over to the 75Ah battery instead of the 18Ah because a couple of times, it turned off because of low-voltage :( .  The 75Ah battery seems to be able to cope with everything so-far. It's hotting up here and the fan runs a little more and there a a lot more birds in Spring cum Summer for it to cope with, but so far, a 75Ah battery is working well.

**Reply by joknjokn** [reply]
What about something like a Bluetti EB3A? You can hook up solar panels to that as well. Maybe even run multiple Birdnet-Pi's with extention cables.

**Reply by unosonic** [reply]
well, if money isn't a problem, you'll find a lot solutions for power supply... I wouldn't recommend a powerstation (with DC/AC converter and other features not needed) for just powering a 5V Pi...

**Reply by joknjokn** [reply]
Sure, depends on the project and money. Anyway, those newer power stations have dedicated usb-outputs (so no unnecessary conversion), and the other outputs are just turned off when not needed = not consuming power. Only loss is general system power. The display can be turned off as well.

**Reply by joknjokn** [reply]
And - for the amount of mAh you get - it's actually not that expensive. Especially not if you count in the included solar capability. :) So again, depends on needs and project.

**Reply by unosonic** [reply]
I know, I have one my own (Ecoflow River Max /w 120W and 140W soalr panels). They're easy to use, that's nice. But for a permanent solar installaton I'd use a dedicated minimal system, eg. battery, solar panel, charge-controller (perhaps with 5V USB included). Outside temperature range is also important, not all batteries like low temp dis/charging. It all depends ;)

**Comment by ExperiMentor** [comment]
Maybe it's obvious to all, but no-one has said it. A Pi 02W will use a LOT less power (therefore any battery will last much longer) than a Pi 4.
